client/src/containers/NavigationItem/Label.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const getCalculatedStyles = ({ sidenav, limit }) => `
+  @media (max-width: ${limit}px) {
+    margin-left: ${sidenav ? '10px' : '0'}
+  }
+`;
+
+const Label = styled.span`
+  margin-left: 10px;
+
+  &:hover {
+    color: white;
+  }
+  
+  ${props => getCalculatedStyles(props)}
+`;
+
+Label.propTypes = {
+  limit: PropTypes.number,
+  sidenav: PropTypes.number,
```

Comment:
I made this to avoid this warning in the console:
![image](https://user-images.githubusercontent.com/26466644/48405325-5f6a8e80-e732-11e8-9c14-2f0a88543994.png)

I am not sure why, but I cannot pass bulean attribute to the dom element... 😞

Reply:
Changed default props (missed last time) 😅